news/2018-06-23-osu-mapping-olympiad-1-results.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #1 Results"
+date: 2018-06-23 16:00:00 +0000
+---
+
+The very first osu! Mapping Olympiad has just concluded, and the results are in! Read on to find out more about our first three Mapping Olympians!
```

Comment:
2nd sentence is super lame - if you dont know what the mapping olympiad is i think this should include something about "If you're interested in 31 mappers competing on 3 exciting songs by Kuba Oms, cYsmix and OISHII and who emerged victorious, continue reading" or _something_ among these lines would probably make it more clear what sort of contest this was? i dont know if this sounds more exciting but i hope you get what i mean